openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -535,6 +543,14 @@ port_INLINE void activity_synchronize_endOfFrame(PORT_RADIOTIMER_WIDTH capturedT
       
       // parse the IEEE802.15.4 header (synchronize, end of frame)
       ieee802154_retrieveHeader(ieee154e_vars.dataReceived,&ieee802514_header);
+
+      //START OF TELEMATICS CODE
+      //if not DAG Root and not synch, store the Key
+  	  if(idmanager_getIsDAGroot()==FALSE && ieee154e_isSynch() == FALSE
+  		 && ieee802514_header.frameType == IEEE154_TYPE_BEACON){
+  		  remote_init(ieee802514_header);

```

Comment:
`remote_init()` is a totally obscure name. Are you initializing your remote controlled airplane?
Please prefix all function name by the file it's in.